google_guest_agent/network/manager/common.go

```diff
@@ -19,13 +19,24 @@ package manager
 import (
 	"fmt"
 	"net"
+	"os/exec"
 
 	"github.com/GoogleCloudPlatform/guest-agent/metadata"
 	"github.com/GoogleCloudPlatform/guest-logging-go/logger"
 )
 
+const (
```

Comment:
Yea only used for tests. I've been considering where best to put them, since I'm planning on using them in other tests as well, so I couldn't figure out a good place to put them out of perhaps a new file with purely shared test definitions.

Reply:
Do we need it in the first place? Can we consider using booleans for that purpose? My feeling is that if we need to define an enum to check a test result we are probably doing it wrong or over complicating the test implementation. Would breaking the test into two or three different test functions resolve or make it simpler?

Reply:
The enum was mainly to avoid having to declare 2 sets of variables -- one to tell the mock functions to return an error, and one to tell the mock function to return true/false. Breaking the test down would be a good idea, though. I've opted for an extra bool value instead.